src/main/java/com/blackducksoftware/integration/hub/alert/digest/DailyItemReader.java

```diff
@@ -85,7 +83,7 @@ private PhoneHomeResponse phoneHome() {
         if (!GlobalProperties.PRODUCT_VERSION_UNKNOWN.equals(productVersion)) {
             final HubServicesFactory hubServicesFactory = globalProperties.createHubServicesFactoryAndLogErrors(logger);
             final PhoneHomeService phoneHomeService = hubServicesFactory.createPhoneHomeService();
-            final PhoneHomeRequestBodyBuilder phoneHomeRequestBodyBuilder = phoneHomeService.createInitialPhoneHomeRequestBodyBuilder(ThirdPartyName.ALERT, productVersion, productVersion);
+            final PhoneHomeRequestBody.Builder phoneHomeRequestBodyBuilder = phoneHomeService.createInitialPhoneHomeRequestBodyBuilder("blackduck-alert", productVersion);
```

Comment:
Food for thought in the future: We could probably check which channels the user has configured and add that as metadata.